File: snippets/csharp/streaming-message_main.cs
// using Microsoft.Agents.Xml.Generated.Models;
// using Microsoft.Agents.Protocol.Xml;

// Create streaming chunk
var chunk = new AgentMessage
{
    AgentId = "agent-456",
    MessageId = "msg-stream-1",
    Contents = new List<AIContent>
    {
        new TextContent { Text = "The weather " }
    }
};

var serializer = new MessageSerializer();
var xmlChunk = serializer.Serialize(chunk);
Console.WriteLine($"Chunk: {xmlChunk}");

File: snippets/python/streaming-message_main.py
# from microsoft.agents.xml.models import AgentMessage, TextContent
# from microsoft.agents.xml.serialization import MessageSerializer
#
# # Create streaming chunk
# chunk = AgentMessage(
#     role="assistant",
#     agent_id="agent-456",
#     message_id="msg-stream-1",
#     contents=[
#         TextContent(text="The weather ")
#     ]
# )
#
# serializer = MessageSerializer()
# xml_chunk = serializer.serialize(chunk)
# print(f"Chunk: {xml_chunk}")